docs/src/pages/demos/dialogs/dialogs.md

```diff
@@ -50,7 +50,7 @@ You can also swap out the transition, the next example uses `Slide`.
 ## Form dialogs
 
 Form dialogs allow users to fill out form fields within a dialog.
-For example, if your site prompts for potential subscribers to fill in their email address, they can fill out the email field and touch 'Submit'
+For example, if your site prompts for potential subscribers to fill in their email address, they can fill out the email field and touch 'Submit'.
```

Comment:
I realize this was a drive-by fix for a missing full stop (period), and that the main goal of this PR is far more important, but I can't help notice the use of "touch", both here and in the subsequent correction. 

I'm wondering what the best device agnostic terminology is? 

"select"?

Reply:
I had thought the same but I'm not sure if select works with 
```
Touch mechanics:
- Choosing an option immediately commits the option and closes the menu
- Touching outside of the dialog, or pressing Back, cancels the action and closes the dialog
```

Reply:
Okay, not just me then. :) Let's discuss it separately so that this can get merged.

Reply:
Good luck with that. I'm moving on 😆.

Reply:
I was able to spot multiple occurrences of the dot issue thanks to Crowdin.